js/roi/ROIManager.js

```diff
@@ -134,9 +134,10 @@ class ROIManager {
         const columns = browser.columnContainer.querySelectorAll('.igv-column')
 
         for (let i = 0; i < columns.length; i++) {
-
+            //span you're looking at
```

Comment:
clean up comments that you added as clarifying notes to yourself before committing